NorthstarDLL/mods/autodownload/moddownloader.cpp

```diff
@@ -0,0 +1,525 @@
+#include "moddownloader.h"
+#include <rapidjson/fwd.h>
+#include <mz_strm_mem.h>
+#include <mz.h>
+#include <mz_strm.h>
+#include <mz_zip.h>
+#include <mz_compat.h>
+#include <thread>
+#include <future>
+#include <bcrypt.h>
+#include <winternl.h>
+#include <fstream>
+
+ModDownloader* g_pModDownloader;
+
+ModDownloader::ModDownloader()
+{
+	spdlog::info("Mod downloaded initialized");
+	modState = {};
+}
+
+size_t writeToString(void* ptr, size_t size, size_t count, void* stream)
```

Comment:
PascalCase for function names

Reply:
Addressed in 7b38a4b60eb550df630506cdf689b94fab99165c